mozetl/experimentsdaily/rollup.py

```diff
@@ -17,6 +17,19 @@ def load_experiments_summary(spark, parquet_path):
     )
 
 
+def add_missing_columns(frame):
+    """
+    to_experiment_profile_day_aggregates() needs the column
+    "scalar_parent_telemetry_os_shutting_down" for EXPERIMENT_FIELD_AGGREGATORS
+    so this function adds it if it's missing from the frame.
+    """
+    from pyspark.sql.functions import lit
+    from pyspark.sql.types import BooleanType
```

Comment:
not sure if I should import these at the top of the file, or here, based on `to_experiment_profile_day_aggregates()`